rust/lance-index/src/scalar/inverted/tokenizer.rs

```diff
@@ -0,0 +1,149 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: Copyright The Lance Authors
+
+use lance_core::{Error, Result};
+use serde::{Deserialize, Serialize};
+use snafu::{location, Location};
+
+/// Tokenizer configs
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct TokenizerConfig {
```

Comment:
Do we store this anywhere? When I run `optimize_indices()`, will it use the same configuration as when I run `create_scalar_index()` with tokenizer configuration?

Reply:
yes we store it in the index, and `optimize_indices` would load the existing index, then it would use this configuration